### DISCUSSION TITLE
Support for collapse-tags props in TagsInput component

### DISCUSSION THREAD

**Discussion by ryuhangyeong** [mantinedev/mantine#7070] [Feature requests]
## Example

![스크린샷 2024-11-04 오전 10 07 08](https://github.com/user-attachments/assets/fb560af3-a88a-4425-baa7-b27b3a7c939d)

https://element-plus.org/en-US/component/select.html#basic-multiple-select

If there are many selected tags, the Input height will become infinitely higher. This doesn't look good. Therefore, I hope props like the example are supported!

![스크린샷 2024-11-04 오전 10 14 59](https://github.com/user-attachments/assets/a020e5f9-7eee-46cd-ab8b-c1bda94c6199)

**Comment by rtivital** [comment]
You can add this logic with Combobox component
